clouddriver-azure/src/main/groovy/com/netflix/spinnaker/clouddriver/azure/resources/servergroup/ops/CreateAzureServerGroupAtomicOperation.groovy

```diff
@@ -56,12 +56,42 @@ class CreateAzureServerGroupAtomicOperation implements AtomicOperation<Map> {
 
       task.updateStatus(BASE_PHASE, "Beginning server group deployment")
 
-      String resourceGroupName = AzureUtilities.getResourceGroupName(description.appName, description.region)
+      String resourceGroupName = AzureUtilities.getResourceGroupName(description.application, description.region)
+      String virtualNetworkName = AzureUtilities.getVirtualNetworkName(resourceGroupName)
+
+      description.credentials.resourceManagerClient.initializeResourceGroupAndVNet(description.credentials, resourceGroupName, description.region)
```

Comment:
We should pass the virtualNetworkName in as a parameter to the initialization method to ensure that the values for the vNet name are the same in both methods.